ui/lib/main.dart

```diff
@@ -0,0 +1,135 @@
+import 'package:cloud_gaming_demo/api/application.dart';
```

Comment:
Can we put a copyright header as we did for the service? Same applies to the other source files